streams/src/test/java/org/apache/kafka/streams/state/internals/ChangeLoggingSessionBytesStoreTest.java

```diff
@@ -63,199 +55,128 @@ public class ChangeLoggingSessionBytesStoreTest {
     @Before
     public void setUp() {
         store = new ChangeLoggingSessionBytesStore(inner);
+        store.init((StateStoreContext) context, store);
     }
 
-    private void init() {
-        EasyMock.expect(context.taskId()).andReturn(taskId).anyTimes();
```

Comment:
I think there is only one method invoked on `context` and we disregard the result from that, so all of these stubs have been removed.

Reply:
Thanks for the additional verification.